### ISSUE TITLE
IDE exception

### ISSUE THREAD

**Issue by patric-eberle** [hsz/idea-gitignore#116]
Since I updated your plugin to version 1.1.1 (at least I can't remember encountering it before), there is an exception every time I close all projects, so the 'welcome' screen is shown and then open an other, existing project.

```
The following Details are given by PhpStorm:
`Directory index is already disposed for Project (Disposed) Intranet (trunk)
java.lang.Throwable
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:115)
	at com.intellij.openapi.roots.impl.DirectoryIndexImpl.b(DirectoryIndexImpl.java:206)
	at com.intellij.openapi.roots.impl.DirectoryIndexImpl.getInfoForFile(DirectoryIndexImpl.java:163)
	at com.intellij.openapi.roots.impl.FileIndexBase.getInfoForFileOrDirectory(FileIndexBase.java:23)
	at com.intellij.openapi.roots.impl.ProjectFileIndexImpl.isInContent(ProjectFileIndexImpl.java:180)
	at com.intellij.psi.search.ProjectAndLibrariesScope.contains(ProjectAndLibrariesScope.java:43)
	at com.intellij.psi.search.GlobalSearchScope.accept(GlobalSearchScope.java:72)
	at com.intellij.util.indexing.FileBasedIndexImpl$14.convert(FileBasedIndexImpl.java:978)
	at com.intellij.util.indexing.FileBasedIndexImpl$14.convert(FileBasedIndexImpl.java:947)
	at com.intellij.util.indexing.FileBasedIndexImpl.a(FileBasedIndexImpl.java:920)
	at com.intellij.util.indexing.FileBasedIndexImpl.a(FileBasedIndexImpl.java:993)
	at com.intellij.util.indexing.FileBasedIndexImpl.getContainingFiles(FileBasedIndexImpl.java:877)
	at com.intellij.psi.search.FileTypeIndex.getFiles(FileTypeIndex.java:44)
	at mobi.hsz.idea.gitignore.IgnoreManager$4.run(IgnoreManager.java:288)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.Alarm$Request$1.run(Alarm.java:351)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:318)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:715)
	at java.awt.EventQueue.access$400(EventQueue.java:82)
	at java.awt.EventQueue$2.run(EventQueue.java:676)
	at java.awt.EventQueue$2.run(EventQueue.java:674)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:86)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:685)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:748)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:577)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:384)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)



java.lang.AssertionError: Already disposed
	at com.intellij.openapi.components.impl.ComponentManagerImpl.getPicoContainer(ComponentManagerImpl.java:281)
	at com.intellij.openapi.project.impl.ProjectImpl.getStateStore(ProjectImpl.java:190)
	at com.intellij.openapi.project.impl.ProjectImpl.getBaseDir(ProjectImpl.java:241)
	at mobi.hsz.idea.gitignore.lang.kind.FossilLanguage.getOuterFile(FossilLanguage.java:74)
	at mobi.hsz.idea.gitignore.IgnoreManager$4.run(IgnoreManager.java:298)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.Alarm$Request$1.run(Alarm.java:351)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:318)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:715)
	at java.awt.EventQueue.access$400(EventQueue.java:82)
	at java.awt.EventQueue$2.run(EventQueue.java:676)
	at java.awt.EventQueue$2.run(EventQueue.java:674)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:86)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:685)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:748)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:577)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:384)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:296)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:211)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:196)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:188)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)`
```

**Comment by hsz** [comment]
Thank you for reporting. The issue has been already reported in #109 and fixed.

It will be pushed with the next release.

**Comment by hsz** [comment]
Fix has ben released just right now with `v1.1.2`

**Comment by patric-eberle** [comment]
Cheers! Thx for the helpful plugin and fast support!
